### PR TITLE
add sponsorship doc to orgs guide

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: content/orgs/sponsorship.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  As a result, `JacquesDerrida` can:
+    - Be added to any/all of the Organization's teams
+    - See the teams he is on
+    - See the members of those teams
+    - Collaborate on and publish any package that his team membership grants him
+      access to
+
+  `JacquesDerrida` cannot:
+    - See all of the Organization's teams
+    - See all members of the Organization
```
Comment (chrisdickinson) [inline content/orgs/sponsorship.md]: This conflates sponsorship with org membership. It breaks down like this:

1. Because `JacquesDerrida` is a `developer` in the org, he can:
  0. Be added to any/all of the Organization's teams,
  1. See the teams he is on,
  2. See the other members of the team,
  3. _See_ the packages (both public and private) those teams grant access to,
2. Because `JacquesDerrida` is sponsored, he can:
  1. Install, publish, and unpublish the private packages to which he has access

File: content/orgs/sponsorship.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  sponsorship of `EmmaGoldman`. This status appears in the UI like this:
+
+  ![not paid](images/sponsorship-red.png)
+
+  As a result, `EmmaGoldman` can:
+    - Be added to any/all of the Organization's teams
+    - See the teams she is on
+    - See the members of those teams
+    - Collaborate on and publish public, scoped or unscoped packages, that her team
+      membership grants her access to
```
Comment (chrisdickinson) [inline content/orgs/sponsorship.md]: `EmmaGoldman` should be able to _see_ the packages, but not install or publish them.

Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: noted!

File: content/orgs/sponsorship.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+    If a user is a subscriber to private packages, this sponsorship scope will
+    trump all other potential sponsorships. If you would like to change this, i.e.
+    offer sponsorship to a user who already has another sponsorship (org or private pkgs),
+    please contact [support@npmjs.com](mailto:support@npmjs.com). 
+
+- ### Not Paid
+
+  When Super Admin, `ag_dubs`, added `EmmaGoldman` to the Organization, `EmmaGoldman`
+  did not subscirbe to private packages nor did she belong to another Organization.
```
Comment (lewiscowper) [inline content/orgs/sponsorship.md]: subscirbe => subscribe

Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: thanks!

File: content/orgs/sponsorship.md (hunk 2 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: @chrisdickinson can you clarify that `EmmaGoldman` can see private packages but not publish to them, but *can* publish publish, scoped or unscoped, packages?

File: content/orgs/sponsorship.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  of the Organization, **paid by another scope**. This appears in the UI like this:
+
+  ![paid, but not by the org](/images/orgs/ships-paidother.png)
+
+  As a result, `CarlSagan` has the same permissions as `JacquesDerrida`, above.
+
+  - #### Paid for by Self
+  
+    As a subscriber to Private Packages, you can understand yourself as "sponsoring
+    yourself". As a result, a subscriber to private packages would have had the same
```
Comment (chrisdickinson) [inline content/orgs/sponsorship.md]: You might change it to `you can understand your sponsorship as "sponsoring yourself"` — the repeated `yourself` here is a bit hard to parse.

File: content/orgs/sponsorship.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  - Because `EmmaGoldman` is a `developer` in the org, they can:
+    - Be added to any/all of the Organization's teams,
+    - See the teams they are on,
+    - See the other members of the team,
+    - _See_ the packages (both public and private) those teams grant access to,
+  - Because `EmmaGoldman` is _not_ sponsored, they _cannot_:
+    - Install, publish, and unpublish the private packages to which they have access
+
+  `EmmaGoldman` can:
+    - Collaborate on and publish public, scoped or unscoped packages, that their team
```
Comment (chrisdickinson) [inline content/orgs/sponsorship.md]: I think `EmmaGoldman` may _currently_ be blocked on publishing scoped packages to orgs without a valid license, but that's a bug and should get fixed pretty quickly when I get back to work.

**Comment by chrisdickinson** [comment]
LGTM

File: content/orgs/sponsorship.md (hunk 4 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: oh good call

**Comment by ashleygwilliams** [comment]
@snopeks @cowperthwait r?

File: content/orgs/sponsorship.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+-->
+
+# Sponsorship
+
+## Sponsorship Types
+
+![org-sponsorship levels](/images/orgs/ships-dashboard.png)
+
+Let's say we have an Organization, `@ag_org`. This Organization
+was created by user `@ag_dubs`, and therefore she is the Super
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: Should terms like “Super Admin” be bolded or ``’d?

File: content/orgs/sponsorship.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+`JacquesDerrida` did not already belong to an organization nor did he have a
+subscription to private packages.
+
+By default, when the Super Admin added him to the Organization, `JacquesDerrida`
+was set as a member of the Organization, **paid by the current organization**.
+This appears in the UI like this:
+
+![paid by current org](/images/orgs/ships-paidcurrent.png)
+
+- Because `JacquesDerrida` is a `developer` in the org, they can:
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: Jacques was referred to as “he” in l. 34; consider using “he”/“him” consistently thruout, or changing l. 34 to match?

File: content/orgs/sponsorship.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+  - Be added to any/all of the Organization's teams,
+  - See the teams they are on,
+  - See the other members of the team,
+  - _See_ the packages (both public and private) those teams grant access to,
+- Because `JacquesDerrida` is sponsored, he can:
+  - Install, publish, and unpublish the private packages to which they have access
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: likewise here; because JD is sponsored _he_ can … … the packed to which _they_ have access.

File: content/orgs/sponsorship.md (hunk 7 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: yar i'm trying to use they for everything. thanks for catching it!

File: content/orgs/sponsorship.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+`JacquesDerrida` cannot:
+  - See all of the Organization's teams
+  - See all members of the Organization
+
+### Paid by Self or Another Organization
+
+- #### Paid for by Another Organization
+  
+  When Super Admin, `@ag_dubs`, added `CarlSagan` to the Organization, `CarlSagan`
+  already belonged to another Organization, (`@nasa-org`, duh).
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: “already belonged to another Organization (`@nasa-org`, duh).” — omits pleonastic comma before open-paren

File: content/orgs/sponsorship.md (hunk 8 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: +1 will change to they

File: content/orgs/sponsorship.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+  of the Organization, **paid by another scope**. This appears in the UI like this:
+
+  ![paid, but not by the org](/images/orgs/ships-paidother.png)
+
+  As a result, `CarlSagan` has the same permissions as `JacquesDerrida`, above.
+
+  - #### Paid for by Self
+  
+    As a subscriber to Private Packages, you can understand your sponsorship as "sponsoring
+    yourself". As a result, a subscriber to private packages would have had the same
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: “you can understand your sponsorship as “sponsoring yourself.”” — I feel like we already had the battle over British vs American period-then-quotation-mark usage, but pushing up my glasses to remind that if we were going with British quotation marking, consistency would also dictate British rules about pluralisation (e.g., a team _are_ vs a team _is_, &c.).

File: content/orgs/sponsorship.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+    default behavior as occured for `CarlSagan`, i.e., the previous sponsorship would
+    trump the possibility of a new Organization sponsorship. A subscriber to private
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: omg and then we could call them _Organisations_

File: content/orgs/sponsorship.md [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+    packages will appear in the Organization dashboard as someone who is `paid` but
+    `not by the current org`. This appears in the UI the same as above:
+
+    ![paid, but not by the org](/images/orgs/ships-paidother.png)
+
+  - #### Changing Sponsorship
+
+    If a user is a subscriber to private packages, this sponsorship scope will
+    trump all other potential sponsorships. If you would like to change this, i.e.
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: “If you would like to change this, i.e., offer sponsorship…” — wraps “i.e.” with commas

File: content/orgs/sponsorship.md (hunk 10 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: oh no

File: content/orgs/sponsorship.md (hunk 11 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: you are drunk with power

File: content/orgs/sponsorship.md [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+    offer sponsorship to a user who already has another sponsorship (org or private pkgs),
+    please contact [support@npmjs.com](mailto:support@npmjs.com). 
+
+- ### Not Paid
+
+  When Super Admin, `ag_dubs`, added `EmmaGoldman` to the Organization, `EmmaGoldman`
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: “When Super Admin, `ag_dubs` added `EmmaGoldman` to the Organization `EmmaGoldman`” — removes comma splices after “`ag_dubs`” and “the Organization `EmmaGoldman`”

File: content/orgs/sponsorship.md (hunk 9 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: +1

File: content/orgs/sponsorship.md (hunk 6 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: hrm. what do you think? i don't have a strong opinion here, but i feel like you are right in that something should happen. we do have a section on it-- would linking to the roles page solve this? (a link would make it visually different)

File: content/orgs/sponsorship.md [hunk 14]
```diff
@@ -0,0 +1,114 @@
...
+  did not subscribe to private packages nor did she belong to another Organization.
+
+  Like `JacquesDerrida`, `EmmaGoldman` was set as **paid by the current organization,
+  `@ag-org`** by default. However, Super Admin, `@ag_dubs` opted to cancel `@ag-org`'s
```
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: “However, as Super Admin, `@ag_dubs` opted to cancel” — or remove comma between “Super Admin” and “`@ag_dubs`”

**Comment by cowperthwait** [comment]
handful of extraordinarily minor copy edits; rest lgtm

File: content/orgs/sponsorship.md (hunk 12 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: +1

File: content/orgs/sponsorship.md (hunk 13 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: wait what? ag-dubs is super admin, so isn't the grammar => ... role, name, .... ?

File: content/orgs/sponsorship.md (hunk 14 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: ah yes ok i agree with this

File: content/orgs/sponsorship.md (hunk 6 above)
Comment (cowperthwait) [inline content/orgs/sponsorship.md]: >would linking to the roles page solve this?

works for me

File: content/orgs/sponsorship.md (hunk 13 above)
Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: ok you are right. i am fixing this

Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: actually i think this makes more sense with no commas. thoughts?

Comment (cowperthwait) [inline content/orgs/sponsorship.md]: Proposed: breaks things into two sentences instead.

“When Super Admin, `ag_dubs` added `EmmaGoldman` to the Organization. `EmmaGoldman` did not subscribe to private packages, nor did she belong to another Organization.”

Comment (cowperthwait) [inline content/orgs/sponsorship.md]: Clarify: the purpose of the sentence is:

Emma didn’t subscribe to private packages or another Org, when she was added by Ashley (who was a superadmin)?

Comment (ashleygwilliams) [inline content/orgs/sponsorship.md]: emma, having not subscribed to private packages or another org, was added to the org by ashley.

Comment (cowperthwait) [inline content/orgs/sponsorship.md]: “When Super Admin `ag_dubs` added `EmmaGoldman` to the Organization, they did not subscribe to private packages nor did they belong to another Organization.”

Removes most of the commas (@ag_dubs was right), changes a few she’s to they’s

File: content/orgs/sponsorship.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+    ![paid, but not by the org](/images/orgs/ships-paidother.png)
+
+  - #### Changing Sponsorship
+
+    If a user is a subscriber to private packages, this sponsorship scope will
+    trump all other potential sponsorships. If you would like to change this, i.e.,
+    offer sponsorship to a user who already has another sponsorship (org or private pkgs),
+    please contact [support@npmjs.com](mailto:support@npmjs.com). 
+
+- ### Not Paid
```
Comment (snopeks) [inline content/orgs/sponsorship.md]: This is going to be really helpful, so many people have questions about how payment works. This looks great.

File: content/orgs/sponsorship.md (hunk 10 above)
Comment (snopeks) [inline content/orgs/sponsorship.md]: The British way is weird, trust me I'm from Canada.

**Comment by ashleygwilliams** [comment]
ZOMG MERGING :dolphin: :sparkles:
